internal/staging/primitives/inclusionproof_test.go

```diff
@@ -0,0 +1,392 @@
+// Copyright (c) 2019-2021 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package primitives
+
+import (
+	"testing"
+
+	"github.com/decred/dcrd/chaincfg/chainhash"
+)
+
+// TestGenerateInclusionProof ensures the expected proofs are produced for
+// known valid leaf values.
+func TestGenerateInclusionProof(t *testing.T) {
```

Comment:
You could add `t.Parallel()` to these tests for consistency since you added it to the rest of the tests in the primitives package so far.

Reply:
Done.